service/frontend/admin_handler_test.go

```diff
@@ -138,6 +139,14 @@ func (s *adminHandlerSuite) SetupTest() {
 		NumHistoryShards: 1,
 	}
 
+	dynamicConfigClient := dynamicconfig.NewMemoryClient()
+	dynamicConfigClient.OverrideValue("frontend.rps", 10)
+	dynamicConfigClient.OverrideValue("matching.rps", 20)
+	dynamicConfigClient.OverrideValue("history.rps", 30)
+	dynamicConfigClient.OverrideValue("limit.maxIDLength", 255)
+	dynamicConfigClient.OverrideValue("history.enableTransitionHistory", true)
```

Comment:
Test more complex configs here. Anything that has slices or maps or is a more complex struct. There are a few of those in `dynamicconfig/constants.go`.